plume_esx/recipe.yaml

```diff
@@ -54,6 +54,11 @@ tasks:
     ref: 89914888d7dd801fb8430aeb386fd423e0f86f46
     dest: ./resources/cron
 
+      - action: download_github
+    src: https://github.com/esx-framework/instance
```

Comment:
What exactly is the purpose of this resource?

Reply:
The water and food values do not reflect their true value. You will notice that these values are not processed on the database. Also, after purchasing the clothes, the esx_clotheshop script will try to connect with esx_property but will not be able to do so. As a result, errors will spew out into the console. I've also included the necessary esx script for players to show signs of intoxication when they consume alcohol. I combined esx_property.sql with plume.sql. But I could not test it. 
There may be some errors. It can be corrected, tested and integrated by you. Thank you, that was my whole goal.